auditbeat/tests/system/test_base.py

```diff
@@ -23,3 +25,24 @@ def test_start_stop(self):
         assert self.log_contains("Setup Beat: auditbeat")
         assert self.log_contains("auditbeat start running")
         assert self.log_contains("auditbeat stopped")
+
+    def test_template(self):
```

Comment:
I think this should have a `@unittest.skipUnless(INTEGRATION_TESTS, "integration test")`?